### PR TITLE
My review

### PR DESCRIPTION
**Pull request by keza681**
## REVIEW PROJECT

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (keza681) [inline index.html:10]: Please remove the `script` at the top of the `HTML` page and put it at the bottom. With this arrangement the code will : 
- Have a perfect look 👍 
- Also, as a convention in coding, we put the `script` line at the bottom in order to allow the engine to very well read the code from top to bottom. Read this article on the topic https://www.w3schools.com/html/html_scripts.asp

File: index.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            type="number"
+            name="phone-numer"
+            id="number"
+            min="10"
+            max="99"
+            class="form-control"
+            placeholder="phone number"
+          />
+        </div>
+        <div class="form-group">
```
Comment (keza681) [inline index.html:59]: Please kindly, add the method, and the action to the `form` in order to allow it to perform different action depending on the method given. for ex : 
`form class="container" action="link of the page(where you want it to go)" method="POST, PUT, DELETE, etc">`
- the **` action`** attribute specify the page, the router of where you want to navigate once you validate this form
- the **` method`** attribute specify the method of the action you put there, for example, you want to delete credentials or to add a given page, or to check for it. that i why it is mandatory to add those details.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        </div>
+      </form>
+    </div>
+    
+
+
+
+    <script  src="js/index.js"></script>
+  </body>
+</html>
```
Comment (keza681) [inline index.html:137]: Please kindly avoid : 
1. too much space between lines of code
2. equalize indentation in the code

**Review by keza681 — COMMENTED** [review]
Hello @keza681  👍 
Very good job done so far, congratulations!
In a short summary
Your project looks great and you did an exceptional work
But there are some issues that still need to work on so that your project will look more interesting than how it is.!
Changes Required 💯 

[REQUIRED ] 
 - In your styling; please before starting to work on the project, configure **Linters** in your local environment. This will help you to discover and to correct small mistakes before committing and pushing the file into GitHub. Hey! No worries, if you forget how to use Linters into Html and Css, please be courageous to have a look at this link https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md
 - Kindly add readme file, to allow the readers of your code to get information before getting through your code.
 - Be kind and follow coding best `practices` while coding to have clean and very standardized codes.
Let me share with you some links which may be helpful for you while going through such practice in the feature.
HTML & CSS best practices : https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md
Watch this short video having the core subject on this topic then come up with the best code practices to enhance the understanding of the viewers.
Inside your CSS file make sure you test your code before submission: https://github.com/microverseinc/linters-config,
Document your code in a professional

Check all the comments under the review.
Cheers and enjoy this and have a happy coding journey!
COURAGE @keza681 👍  👍  👍

**Review by elmar8287 — APPROVED** [review]
Hi @keza681 , 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**